.github/workflows/bicep_workflow.yml

```diff
@@ -0,0 +1,76 @@
+name: Deploy Infrastructure using Bicep
+concurrency: bicep-deploy
+
+on:
+  push:
+    branches:
+      - feat/FN-72/integrate-build-pipeline
```

Comment:
Obviously I'll need to update this!
Note that the current plan is to reflect the behaviour for `dev` and have a different branch for infrastructure deployments, but I have a card to make it integrated and nicer.